modules/text_server_adv/text_server_adv.cpp

```diff
@@ -1830,6 +1830,14 @@ bool TextServerAdvanced::_ensure_cache_for_size(FontAdvanced *p_font_data, const
 			p_font_data->face_init = true;
 		}
 
+#ifdef MACOS_ENABLED
+		if (p_font_data->font_name == ".Apple Color Emoji UI") {
```

Comment:
Should we do like https://github.com/linebender/vello/blob/244ed01801816b99c99adcbbedc3612962919040/vello/src/scene/bitmap.rs#L202-L214 and only hardcode it if `baseline_offset` is 0, in case Apple fixes their Color Emoji UI font?

Reply:
Not sure if it's worth doing for every glyph, but I guess we can check one when font is loaded.